### ISSUE TITLE
Operator Linebreak in relation to Ternary

### ISSUE THREAD

**Issue by OzoneTank** [smashingboxes/eslint-config-smashing-boxes#7]
## Operator Linebreak
I'm proposing that we change the `operator-linebreak` setting currently found https://github.com/smashingboxes/eslint-config-smashing-boxes/blob/master/defaults.js#L245 to include `"overrides": { "?": "before", ":": "before" }`

See the documentation https://eslint.org/docs/rules/operator-linebreak and the specific setting https://eslint.org/docs/rules/operator-linebreak#overrides

This proposal would be following this proposal https://github.com/smashingboxes/eslint-config-smashing-boxes/issues/6

**Comment by leeallen337** [comment]
I'm on the fence about this for the following reason. For the most part I've grown to be ambivalent to operators being at the end of the line or the beginning of a new line. However, I think it makes sense to have all the operators follow the same pattern, either the beginning or the end of a line. Thoughts?

**Comment by OzoneTank** [comment]
I agree to keep thing consistent. My only reason for wanting to put the ternary operators at the beginning is for readability's sake. If I see a nested multiline ternary I would be more comfortable reading it with the operators at the beginning.

**Comment by leeallen337** [comment]
It's always been my opinion that having the operators at the beginning of a line, regardless of whether it's a ternary or a plus-sign, etc. and regardless of nesting, is easier to read 🤷‍♂️ 

Then all your operators line up

**Comment by OzoneTank** [comment]
Ahh yes, I like that better actually.

**Comment by OzoneTank** [comment]
Note: we need to talk about about operators being `before` or `after` and we need to resolve https://github.com/smashingboxes/eslint-config-smashing-boxes/issues/6 before we can move further with this

**Comment by leeallen337** [comment]
So I was just testing this out some more and this also impacts things like `||` and `&&` so with the setting `before` it would require changing this from
```javascript
{ props.truthy &&
  <Component />
}
```
to the following
```javascript
{ props.truthy
  &&  <Component />
}
```
